bot/validations.ts

```diff
@@ -562,11 +585,11 @@ const validateParams = async (ctx, paramNumber, errOutputString) => {
     return params.slice(1);
   } catch (error) {
     logger.error(error);
-    return false;
+    return null;
```

Comment:
@Mersho this is great; maybe this change (and the subsequente change over the callers of this func) should go in its own commit